site/pages/docs/versions/dwnld-en.hbs

```diff
@@ -108,6 +107,17 @@
 				<li>IE 11</li>
 				<li>Edge 17</li>
 			</ul>
+			<p class="mrgn-bttm-xl">As per <a href="http://wet-boew.github.io/wet-boew-documentation/decision/2.html">Design decision 2: Browser supported</a></p>
+		</section>
+		<section>
+			<h4 id="v4028.2">v4.0.28.2</h4>
+			<p><a href="v4.0/v4.0.28-en.html#v4.0.28.2">Release notes<span class="wb-inv"> - v4.0.28.2</span></a></p>
+			<p><a href="https://github.com/wet-boew/wet-boew/releases/download/patch-v4.0.21-to-v4.0.29/patch-for-v4.0.21-to-v4.0.29.zip">Download files<span class="wb-inv"> - v4.0.28.2</span></a> (Read the <a href="v4.0/patch-for-v4.0.21-to-v4.0.29-en.html">Instructions on applying the feed patch for WET-BOEW and related themes</a>)</p>
+			<h5>Patch applies to the following themes:</h5>
+			<ul class="mrgn-bttm-xl">
+				<li>v4.0.28.2 - Web Experience Toolkit (WET)</li>
+				<li>v4.0.28.2 - Canada.ca (GCWeb)</li>
```

Comment:
Can you remove the version number for each listed themes? It is not really necessary as the section is specific to a version.